Made the BIDS example more general for the audience

jcohenadad opened the pull request:
Made the BIDS example more general for the audience

jcohenadad committed:
Update data.rst

Made the BIDS example more general for the audience

jcohenadad committed:
Fixed typo in URL

ahill187 commented on docs/source/data.rst:
```diff
@@ -14,23 +14,20 @@ An example of this organization is shown below:
     └── derivatives
         └── labels
-            └── sub-siteX01
+            └── sub-03
```
Do each of the datasets need to have a labels folder? If so, it might be an idea to include `sub-01` and `sub-02` here as well?

jcohenadad replied:
> Do each of the datasets need to have a labels folder? If so, it might be an idea to include `sub-01` and `sub-02` here as well?

sub-01 and sub-02 are supposed to have one as well. It is implied here (for the sake of clarity).

ahill187 reviewed (commented):
I'm not sure if this is beyond the scope of the pull request, but it might be worth explaining a bit about what is in these folders/files and what `sub-01` means, for example?

jcohenadad committed:
Explicit what sub-01 meant

Addressing https://github.com/ivadomed/ivadomed/pull/597#pullrequestreview-561450940

jcohenadad commented:
> I'm not sure if this is beyond the scope of the pull request, but it might be worth explaining a bit about what is in these folders/files and what `sub-01` means, for example?

thank you, does https://github.com/ivadomed/ivadomed/pull/597/commits/30f826f3ce607ce65e0ec6e1849b37a133a8c2f2 help?

ahill187 commented:
Yes, https://github.com/ivadomed/ivadomed/pull/597/commits/30f826f3ce607ce65e0ec6e1849b37a133a8c2f2 helps clarify what is in the folders.

ahill187 committed:
Merge branch 'master' into jca/update-data-bids

ahill187 committed:
Merge branch 'master' into jca/update-data-bids

ahill187 committed:
Merge branch 'master' into jca/update-data-bids

ahill187 committed:
Merge branch 'master' into jca/update-data-bids
